src/PowerShellEditorServices/Services/PowerShellContext/Session/PSReadLinePromptContext.cs

```diff
@@ -22,26 +22,15 @@ internal class PSReadLinePromptContext : IPromptContext
             "..",
             "..",
             "..",
+#if TEST
```

Comment:
OHH @rjmholt this won't work because we need it defined during test compilation but for this project. Ugh how on earth do we do that?

Reply:
Found the answer https://stackoverflow.com/questions/59249847/passing-define-constants-to-dotnet-build-kills-target-framework-symbols